src/app/(dashboard)/task/create/page.tsx

```diff
@@ -0,0 +1,143 @@
+            <Text asChild size="2" weight="bold">
+              <label htmlFor="category">Category</label>
+            </Text>
+            <TextField.Root
+              id="category"
+              variant="surface"
+              placeholder="Cleaning"
+            />
+          </Flex>
+
+          <Flex direction="column" gap="2">
+            <Flex align="center" justify="between" my="4">
+              <Heading as="h3" size="4" trim="both">
+                Where does the task need to be done?
+              </Heading>
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="address_line_1">Address Line 1</label>
+              </Text>
+              <TextField.Root
+                id="address_line_1"
+                variant="surface"
+                placeholder="123 George St"
+              />
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="address_line_2">Address Line 2</label>
+              </Text>
+              <TextField.Root
+                id="address_line_2"
+                variant="surface"
+                placeholder=""
+              />
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="suburb">Suburb</label>
+              </Text>
+              <TextField.Root
+                id="suburb"
+                variant="surface"
+                placeholder="Sydney"
+              />
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="state">State</label>
+              </Text>
+              <TextField.Root id="state" variant="surface" placeholder="NSW" />
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="postcode">Postcode</label>
+              </Text>
+              <TextField.Root
+                id="postcode"
+                variant="surface"
+                placeholder="2000"
+              />
+            </Flex>
+          </Flex>
+
+          <Flex direction="column" gap="2">
+            <Flex align="center" justify="between" my="4">
+              <Heading as="h3" size="4" trim="both">
+                When would you like your task done?
+              </Heading>
+            </Flex>
+
+            <Flex direction="column" gap="2">
+              <Text asChild size="2" weight="bold">
+                <label htmlFor="address_line_1">Preferred Time</label>
+              </Text>
+              <TextField.Root
+                type="date"
+                id="address_line_1"
+                variant="surface"
+                placeholder="123 George St"
+              />
```

Comment:
Need to look into a calendar component. For now, we'll use the janky browser date input

Reply:
I used the `react-date-range` package for one of my projects and its an absolute beauty.

Reply:
Cheers Promie! Will take a look. :smile: